transform/defaults.go

```diff
@@ -20,7 +20,11 @@ import (
 	"github.com/compose-spec/compose-go/v2/tree"
 )
 
-var defaultValues = map[tree.Path]transformFunc{}
+// DefaultValues contains the default value transformers for compose fields
+var DefaultValues = map[tree.Path]TransformFunc{}
+
+// For backward compatibility
+var defaultValues = DefaultValues
```

Comment:
not needed, as a non-public field there's no backward compatibility to be addressed here